@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/core/fn_coreGetWCAs.sqf

```diff
@@ -162,6 +162,16 @@ if (_heli getVariable "fza_ah64_e2_fire") then {
 } else {
     [_activeWarn, "ENGINE 2 FIRE"] call fza_wca_fnc_wcaDelWarning;
 };
+//--Aft Deck fire
+if (_heli getVariable "fza_ah64_aft_deck_fire") then {
+    /*([_heli, _activeWarn, "AFT DECK FIRE", "DECK FIRE", FIRE_PRIORITY, "fza_ah64_aft_deck_fire", 3] call fza_wca_fnc_wcaAddWarning)
+        params ["_wcaAddWarning"];*/
+    _wcas pushBack [WCA_WARNING, "AFT DECK FIRE", "DECK FIRE"];
+} else {
```

Comment:
Remove the commented areas if not required anymore, and the else if not required anymore